index.qmd

```diff
@@ -24,42 +24,50 @@ format:
   RSEs have organised themselves in various national societies, in Germany this is de-RSE e.V \cite{derseev}.
 
-While there is overlap between all these fields, we can summarise that RSEs focus on the creation of software in order to facilitate research using their SE skills.
-Research data manageners focus on the data that is generated during the research process and provide infrastructure for better provisioning of the data.
-Data Scientists focus on gaining insight from the data that is generated somewhere .
+- **Research Infrastructure Engineers / High Performance Computing (HPC) Engineers** support the running of research computing infrastructure that is vital to support the demands of the modern research lifecycle. While people have undertaken such roles for many years, they have often been embedded within or been a part of wider system adminstration or management roles. With the expansion of digital research and the associated computing infrastructure, we are increasingly seeing dedicated HPC and research computing infrastrcture-focused roles that are based within teams or groups dedicated to supporting research computing. These individuals design, deploy and maintain research computing clusters and provide software and systems expertise to help researchers undertake their computations in an efficient and effective manner.
 
-In this paper we focus on the competencies of RSEs and the data scientists.
+While there is overlap between all these fields, we can summarise that RSEs focus on the creation of software in order to facilitate research using their Software Engineering (SE) skills.
+Research Data Managers focus on the data that is generated and/or used during the research process and provide infrastructure for better provisioning of the data.
+Data Scientists focus on gaining insight from the data that is generated somewhere.
+Research Infrastructure Engineers focus on provisioning and running the computing platforms that software and data experts often use to support their work.
+
+In this paper we focus on the competencies of RSEs and Data Scientists and compare and contrast the skills and activities of these two groups.
 (WHY?!)
-Similar to Research Software Engineering (RSE), Data Science (DS) is a cross-cutting field that focuses on a special area within the sciences or data oriented businesses.
 
-The major computing organizations ACM and IEEE published a joint recommended computing curriculum which already included data science: the Computing Curriculum 2020 [@CC2020] already lists DS as one of the special cases for computing competences.
+We focus our comparison in this paper on these two groups for three reasons:
```

Comment:
There was a note in the text "WHY?!" in relation to the comparison between these two fields - I've tried to address this with the addition of the three points below.